addons/default/visiosoft/advs-module/src/AdvsModulePlugin.php

```diff
@@ -115,6 +115,12 @@ function ($name) {
                     );
                 }
             ),
+	        new \Twig_SimpleFunction(
+		        'dateFormat',
+		        function ($date, $functionName) {
+			        return $date->{$functionName}();
+		        }
+	        )
```

Comment:
you don't need this function you can simply use this for example `adv.created_at.diffForHumans`